src/main/java/com/raoulvdberge/refinedstorage/apiimpl/storage/disk/StorageDiskManager.java

```diff
@@ -40,6 +40,12 @@ public IStorageDisk get(UUID id) {
         return disks.get(id);
     }
 
+    @Nullable
+    public UUID getUuid(IStorageDisk disk) {
+        // Return random UUID for now
+        return UUID.randomUUID();
```

Comment:
???

You need to add a `getId` method to `IStorageDisk`, and change all the implementers accordingly.

Reply:
If you want, I can do this too. Just let me know.

Reply:
I will do it myself.